libpod/container_inspect.go

```diff
@@ -401,3 +456,56 @@ func parseMountOptionsForInspect(options []string, mount *InspectMount) {
 	mount.Mode = zZ
 	mount.Options = otherOpts
 }
+
+// Generate the InspectContainerConfig struct for the Config field of Inspect.
+func (c *Container) generateInspectContainerConfig(spec *spec.Spec) (*InspectContainerConfig, error) {
+	ctrConfig := new(InspectContainerConfig)
+
+	ctrConfig.Hostname = c.Hostname()
+	ctrConfig.User = c.config.User
```

Comment:
This looks like it used to be spec.Process.User.  Is it OK to convert or should you take it from spec instead if it's not empty?

Reply:
This appears to be better - Docker prints the username, not the UID/GID as the Spec includes.

Reply:
Shrug, I've always been partial to numeric id's for stuff like this over strings, if no one else shares that same personal twitch, then it's fine by me.